.github/workflows/tests.yml

```diff
@@ -0,0 +1,29 @@
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/checkout@v2
+        with:
+          repository: alphagov/govuk-content-schemas
+          ref: deployed-to-production
+          path: govuk-content-schemas
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: .ruby-version
+      - uses: actions/cache@v1
+        with:
+          path: vendor/bundle
+          key: ${{ runner.os }}-gems-${{ hashFiles('**/Gemfile.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-gems-
+
+      - run: |
+          bundle config path vendor/bundle
+          bundle install --jobs 4 --retry 3
+      - run: bundle exec rake
+        env:
+          GOVUK_CONTENT_SCHEMAS_PATH: ./govuk-content-schemas
+      - run: bundle exec rubocop
```

Comment:
I wondered whether we were missing sass linting too - but looks like it was never set-up on this repo 😞